finalProject.css

```diff
@@ -79,10 +81,32 @@ h2 {
     text-decoration-line: underline;
     margin: 14px;
     align-items: start;
+    flex-direction: column;
 }
 
+/* button edits */
+
 button {
     display: flex;
+    color: brown;
+    border-radius: 10px;
+    background-color: pink;
+}
+
+#add-tie {
+    margin-top: 40px;
+}
+
+#add-shirt {
+    margin-top: 96px;
+}
+
+#add-coatslack {
+    margin-top: 94px;
+}
+
+#add-shoes {
+    margin-top: 82px;

```

Comment:
In general, it is better to use classes than ids for styling. Ids are great for js actions but can slow down your CSS. I would probably use one class here and make the distance between each button the same.

Reply:
I agree. I originally used one class and deviated from that course of action as the spacing was never correct. Would multiple classes be okay as well?